### PR TITLE
FileManager: describe api

### PR DESCRIPTION
**Pull request by AnnaKondratova**

### CODE REVIEW

File: api-reference/10 UI Widgets/dxFileManager/1 Configuration/onErrorOccurred.md [hunk 1]
```diff
@@ -6,28 +6,54 @@ EventForAction: dxFileManager.errorOccurred
...
-<!-- Description goes here -->
+Model data. Available only if you use Knockout.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+
+The following error codes are supported:
+
+    NoAccess = 0,
```
Comment (RomanTsukanov) [inline api-reference/10 UI Widgets/dxFileManager/1 Configuration/onErrorOccurred.md]: I think this information is more suitable in the description of the `errorCode` field.

Comment (AnnaKondratova) [reply api-reference/10 UI Widgets/dxFileManager/1 Configuration/onErrorOccurred.md]: updated

File: api-reference/10 UI Widgets/dxFileManager/1 Configuration/onErrorOccurred.md [hunk 2]
```diff
@@ -6,28 +6,59 @@ EventForAction: dxFileManager.errorOccurred
...
 
 ##### field(e.element): dxElement
-<!-- Description goes here -->
+#include common-ref-elementparam with { element: "widget" }
 
 ##### field(e.errorCode): Number
-<!-- Description goes here -->
+The error code. The following error codes are supported:
+
+- NoAccess = 0,
```
Comment (RomanTsukanov) [inline api-reference/10 UI Widgets/dxFileManager/1 Configuration/onErrorOccurred.md]: Commas are not required

Comment (AnnaKondratova) [reply api-reference/10 UI Widgets/dxFileManager/1 Configuration/onErrorOccurred.md]: ok

**Review by RomanTsukanov — APPROVED** [review]

**Pull request merged**